django_ilmoitin/models.py

```diff
@@ -19,9 +20,35 @@ class NotificationTemplate(TranslatableModel):
         verbose_name=_("type"),
         unique=True,
         db_column="type",
+        default="",
         choices=list(notifications.registry.items()),
     )
 
+    from_email = models.EmailField(
+        verbose_name=_("From email"), max_length=100, default="dev@hel.fi"
```

Comment:
could you make the default come from `settings.DEFAULT_FROM_EMAIL` and mention this in the README? would probably be a cleaner approach for this.